File: app/app/src/main/java/com/resourceit/app/tools/AppDatabase.java
package com.resourceit.app.tools;

import com.resourceit.app.dao.LoginDao;
import com.resourceit.app.models.LoginModel;

import androidx.room.Database;
import androidx.room.RoomDatabase;

@Database(entities = {LoginModel.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {
    public abstract LoginDao loginDao();
}

File: BankApp/app/src/main/java/br/com/rms/bankapp/data/local/database/AppDatabase.kt
package br.com.rms.bankapp.data.local.database

import androidx.room.Database
import androidx.room.RoomDatabase
import br.com.rms.bankapp.data.local.database.dao.AccountDao
import br.com.rms.bankapp.data.local.database.dao.StatementDao
import br.com.rms.bankapp.data.local.database.dao.UserDao
import br.com.rms.bankapp.data.local.database.entity.Account
import br.com.rms.bankapp.data.local.database.entity.Statement
import br.com.rms.bankapp.data.local.database.entity.User

const val DATABASE_NAME = "bankapp.db"

@Database(entities = [User::class, Account::class, Statement::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {

    abstract fun userDao(): UserDao

    abstract fun accountDao(): AccountDao

    abstract fun statementDao(): StatementDao
}